tests/expressions/models.py

```diff
@@ -84,3 +84,18 @@ class UUID(models.Model):
 
     def __str__(self):
         return "%s" % self.uuid
+
```

Comment:
I've just included the models/tests from the initial bug report: I tried to remap the query onto an existing model structure from this file, but couldn't really find a clean match.